client/dashboard/data/site-logs.ts

```diff
@@ -0,0 +1,110 @@
+import wpcom from 'calypso/lib/wp';
+
+export interface SiteLogsAPIResponse {
+	message: string;
+	data: {
+		total_results: number | { value: number; relation: string };
+		logs: ( PHPLogFromEndpoint | ServerLogFromEndpoint )[];
+	};
+}
+
+export interface PHPLogFromEndpoint {
+	timestamp: string;
+	severity: 'User' | 'Warning' | 'Deprecated' | 'Fatal error';
+	message: string;
+	kind: string;
+	name: string;
+	file: string;
+	line: number;
+	atomic_site_id: number;
+}
+
+export interface PHPLog extends Omit< PHPLogFromEndpoint, 'atomic_site_id' > {
+	id: string;
+}
+
+export enum LogType {
```

Comment:
We'd prefer to use const object instead of enum. See p1754323932129889-slack-C0982082MSR.